admin/tools/docker/integration-tests/localhost/run-multinode-tests.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+# LSST Data Management System
+# Copyright 2014 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+
+# Launch Qserv integration tests on one Docker host
+# using multiple Docker containers
+
+# @author  Fabrice Jammes, IN2P3/SLAC
+
+set -x
+set -e
+
+. ./env.sh
+
+
+usage() {
+  cat << EOD
+
+  Usage: $(basename "$0") [options]
+
+  Available options:
+    -h          this message
+    -C          Rebuild the images from scratch
+
+  Launch Qserv integration tests on one Docker host 
+
+EOD
+}
+
+# get the options
+while getopts hC c ; do
+    case $c in
+            h) usage ; exit 0 ;;
```

Comment:
why is `-h` a normal exit status while `-?` is not?  Could these be combined into a single case as `h | \? )`?